2.0/s2i/bin/run

```diff
@@ -10,6 +10,8 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_vars=$(cgroup-limits.py) ; export $export_vars
```

Comment:
Not critical, but was something I wanted to have mentioned in the original PR to sti-base but had no chance to review before it merged ;)

If we'll ever fix it, better do it sooner than later, before this name will be all around every sti-* image.